src/Microsoft.Data.SqlClient/src/Interop/Windows/Interop.Errors.cs

```diff
@@ -0,0 +1,15 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+internal partial class Interop
+{
+    // https://msdn.microsoft.com/en-us/library/windows/desktop/ms681382.aspx
+    internal partial class Errors
```

Comment:
I see all the other interop file changes as moves, but why were this file and GetFileType_SafeHandle.cs created new?

Reply:
I'm pretty sure this is because the original `netcore/src/Common/src/Interop/Windows/Interop.Errors.cs` and `netcore/src/Common/src/CoreLib/Interop/Windows/Kernel32/Interop.GetFileType_SafeHandle.cs` need to be deleted.

Reply:
Yep, it looks like either I forgot to remove these files or I didn't pick up the deletion in the commit. Easy fix :+1: